CONFIGDOC.md

```diff
@@ -67,7 +67,9 @@ You can define several axes in the **Configuration Matrix** section. For example
 
 ![axis_version_testtag](https://user-images.githubusercontent.com/48831250/76800736-aa733680-67aa-11ea-86a7-a984d5795e11.png)
 
-2) In the **Build Environment** section, select **Use MATLAB version** and then specify the locations where MATLAB is installed using the user-defined axis name. In this example, **$VERSION** in the **MATLAB root** will be replaced by one axis value per build iteration.
+2) In the **Build Environment** section, select **Use MATLAB version** and then specify the locations where MATLAB is installed using the user-defined axis name. (You can skip this step if MATLAB has already been added to the path on the build agents.)

```

Comment:
In the case of the matrix build, when looping over different versions of MATLAB I think it is less likely that they will skip this step. Each build agent can only have one matlab on the PATH, and when the matrix build loops over the parameter it will need to add that specific version to the top of the path.

Reply:
Thanks for clarifying. I removed the parenthetical information.